src/bindings/c/include/openvino/c/ov_core.h

```diff
@@ -215,6 +215,16 @@ ov_core_compile_model(const ov_core_t* core,
                       ov_compiled_model_t** compiled_model,
                       ...);
 
+/**
+ * @brief Adds an extension to the core.
+ * @ingroup ov_core_c_api
+ * @param core A pointer to the ov_core_t instance.
+ * @param library_path Path to an extension.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e)
+ov_core_add_extension(const ov_core_t* core, const char* library_path);
```

Comment:
Could you also add this API into doc?
https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/c/docs/api_overview.md

Reply:
I added it. However, if i try to use `clang-format` for this file, it changes a lot of stuff in it, so I haven't run it for this file. I am not sure if there is a different process for this specific file `api_overview.md`. clang-format ran as expected for other files.

Reply:
I think we needn't run clang-format for this document file, right?